packages/esm-patient-search-app/src/mpi/utils.ts

```diff
@@ -0,0 +1,49 @@
+import { capitalize } from 'lodash-es';
+import { type SearchedPatient } from '../types';
+import { getCoreTranslation } from '@openmrs/esm-framework';
+export function inferModeFromSearchParams(searchParams: URLSearchParams) {
+  return searchParams.get('mode')?.toLowerCase() === 'external' ? 'external' : 'internal';
```

Comment:
These should probably be `mpi` and just no value, which can be interpretted as `local` or whatever.